lib/parsers/javascript.js

```diff
@@ -15,27 +14,25 @@ var babel = require('babel-core'),
  */
 function parseJavaScript(data) {
   var results = [];
-  var ast = babel.parse(data.source, {
+  var transformed = babel.transform(data.source, {
     code: false,
-    stage: 0,
-    locations: true,
-    ranges: true
+    presets: ['es2015', 'stage-0', 'react']
```

Comment:
You only need to parse the ast correct? You can just use babylon for that and use the seperate babel-traverse package.

Example of what I did with babylon options: https://github.com/cst/cst/blob/master/src/Parser.js#L73-L94